crossbeam-channel/tests/list.rs

```diff
@@ -580,3 +580,84 @@ fn channel_through_channel() {
     })
     .unwrap();
 }
+
+#[test]
```

Comment:
maybe better off writing a test for possible race condition, where a sending thread disconnect/reconnect/send(T) successively , and the receiver thread may/may not see disconnected, or actual message.